bundle/src/main/java/com/adobe/acs/commons/dam/impl/CopyAssetPublishUrlFeature.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2017 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package com.adobe.acs.commons.dam.impl;
+
+import org.apache.felix.scr.annotations.*;
+import org.apache.jackrabbit.oak.commons.PropertiesUtil;
+import org.apache.sling.featureflags.ExecutionContext;
+import org.apache.sling.featureflags.Feature;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+@Service(value = Feature.class)
+@Properties({@Property(name = "service.ranking", intValue = 200)})
+@Component(label = CopyAssetPublishUrlFeature.LABEL, description = CopyAssetPublishUrlFeature.DESCRIPTION, metatype = true)
+public class CopyAssetPublishUrlFeature implements Feature {
```

Comment:
Can't we just use the Configured Feature for this and not have any code?

Reply:
I did not get it. Which feature you are talking about ?

Reply:
There's a generic component in Sling which allows for features to be defined purely via configuration. The PID for this is `org.apache.sling.featureflags.impl.ConfiguredFeature`, so you create a configuration like `org.apache.sling.featureflags.impl.ConfiguredFeature-myfeature`, set the `enabled` property to true and the `name` property to the feature name.